backend/study/queries.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+from graphene import Field, List, ObjectType, ResolveInfo, Int
+
+from django.db.models import QuerySet
+
+from study.models import Study
+from study.types.StudyType import StudyType
+from main.models import MRPermissionTypes
+
+
+class StudyQuery(ObjectType):
+    study = Field(
+        StudyType,
+        id=Int(),
```

Comment:
For IDs, you can use GraphQL's `ID` type (import from `graphene`). Also, this argument should be marked as required. (My query fails without it.)


```suggestion
        id=ID(required=True),
```